### ISSUE TITLE
When playing local list MASS plays Spotify instead

### ISSUE THREAD

**Issue by AndreasBVB** [music-assistant/hass-music-assistant#1048]
### What version of Music Assistant has the issue?

2022.8.4

### The problem

When playing a pure local list MASS randomly search for equivalent Spotify sources and play instead. From time to time it plays the wrong song.

When playing a Spotify/Local mixed playlist, MASS plays Spotify as much as possble, also with occasion mismatch.

### How to reproduce

Can't tell when it chooses to do this. Playing a mixed list might help.

### What version of Home Assistant Core are your running

2022.9.2

### What type of installation are you running?

Home Assistant Container

### On what type of hardware are you running?

Linux

**Comment by AndreasBVB** [comment]
Example:
![Screenshot_20221226-011416__01__01](https://user-images.githubusercontent.com/89738769/209512105-4901abcd-258d-44bd-b946-48f7ba54420d.jpg)

**Comment by AndreasBVB** [comment]
Here is the Ed Sheeran local MP3 song featuring Taylor Swift. However I play it, MASS always plays a different Spotify Ed Sheeran solo remix of it. There is no way I can play the local version featuring Taylor.
https://user-images.githubusercontent.com/89738769/209966885-fb8c20ad-dd54-41d1-8128-0d8a0cc68456.mp4

Here is the MP3 info extracted from PowerAmp in Android:
![Screenshot_20221229-083302__01](https://user-images.githubusercontent.com/89738769/209967160-867ca425-d69d-4960-948f-4197065ef264.jpg)

![Screenshot_20221229-083314__01](https://user-images.githubusercontent.com/89738769/209967173-93f573f3-eded-4553-a0ca-a207f46404ef.jpg)

**Comment by erkr** [comment]
It is by design that MA looks for the best quality of each track across the music sources configured.
Pity that a wrong track is selected. I expect this is due the mp3 tags that are provided by your local file are matching

**Comment by AndreasBVB** [comment]
> It is by design that MA looks for the best quality of each track across the music sources configured. Pity that a wrong track is selected. I expect this is due the mp3 tags that are provided by your local file are matching

Would rather have an option to turn off this smartness because it might be wrong about both quality and tags 🤦

**Comment by erkr** [comment]
Yes that is a valid enhancement request. I can move this to the feature request discussion thread if you like

**Comment by AndreasBVB** [comment]
> 

Thank you so much please 🙏
